modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must specify the projects associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same project as the `ServiceMeshControlPlane` resource and name it `default`.
+
+[WARNING]
+====
+If Container Network Interface (CNI) is enabled, manual sidecar injection will work, but pods will not be able to communicate with the control plane unless those pods are a part of the `ServiceMeshMemberRoll` resource.
+====
+
+[NOTE]
+====
+The member projects are only updated if the {ProductShortName} control plane installation succeeds.
+====
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll`.
+
+* `ServiceMeshMemberRoll` resources are reconciled in response to the following events:
+
+** The `ServiceMeshMemberRoll` is created, updated, or deleted
+** The `ServiceMeshControlPlane` resource in the project containing the `ServiceMeshMemberRoll` is created or updated
+** A project listed in the `ServiceMeshMemberRoll` is created or deleted
+
+The `ServiceMeshMemberRoll` is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+[id="ossm-member-roll-console_{context}"]
+== Configure the member roll from the the {product-title} web console
+
+This example joins the Bookinfo project to the {ProductShortName} from the web console.
```

Comment:
This ends up rendering as "... joins the Bookinfo project to the Service Mesh from the web console". Given how there can be multiple service meshes, the sentence is somewhat misleading as it makes it seem as though there's only one mesh. Plus, the product name is used here - if it changes, this sentence may no longer make sense.

Reply:
I've changed this to read:
"Follow this procedure to add the Bookinfo project to the {ProductShortName} member roll by using the web console."